hikari/api/rest.py

```diff
@@ -2160,28 +2501,116 @@ async def create_guild_text_channel(
         category: undefined.UndefinedOr[snowflakes.SnowflakeishOr[channels.GuildCategory]] = undefined.UNDEFINED,
         reason: undefined.UndefinedOr[str] = undefined.UNDEFINED,
     ) -> channels.GuildTextChannel:
-        ...
-
-    @abc.abstractmethod
-    async def create_guild_news_channel(
-        self,
-        guild: snowflakes.SnowflakeishOr[guilds.PartialGuild],
-        name: str,
-        *,
-        position: undefined.UndefinedOr[int] = undefined.UNDEFINED,
-        topic: undefined.UndefinedOr[str] = undefined.UNDEFINED,
-        nsfw: undefined.UndefinedOr[bool] = undefined.UNDEFINED,
-        rate_limit_per_user: undefined.UndefinedOr[int] = undefined.UNDEFINED,
-        permission_overwrites: undefined.UndefinedOr[
-            typing.Sequence[channels.PermissionOverwrite]
-        ] = undefined.UNDEFINED,
-        category: undefined.UndefinedOr[snowflakes.SnowflakeishOr[channels.GuildCategory]] = undefined.UNDEFINED,
-        reason: undefined.UndefinedOr[str] = undefined.UNDEFINED,
-    ) -> channels.GuildNewsChannel:
-        ...
+        """Create a text channel in a guild.
 
-    @abc.abstractmethod
-    async def create_guild_voice_channel(
+        Parameters
+        ----------
+        guild : hikari.snowflakes.SnowflakeishOr[hikari.guilds.PartialGuild]
+            The guild to create the channel in. This may be a
+            guild object or the ID of an existing guild.
+        name : builtins.str
+            The channels name.
+        position : hikari.undefined.UndefinedOr[builtins.int]
+            If provided, the position of the channel (relative to the
+            category, if any).
+        topic : hikari.undefined.UndefinedOr[builtins.str]
+            If provided, the channels topic.
+        nsfw : hikari.undefined.UndefinedOr[builtins.bool]
+            If provided, whether to mark the channel as NSFW.
+        rate_limit_per_user : hikari.undefined.UndefinedOr[builtins.int]
+            If provided, the rate limit per user.
+        permission_overwrites : hikari.undefined.UndefinedOr[typing.Sequence[hikari.channels.PermissionOverwrite]]
+            If provided, the permission overwrites for the channel.
+        category : hikari.undefined.UndefinedOr[hikari.snowflakes.SnowflakeishOr[hikari.channels.GuildCategory]]
+            The category to create the channel under. This may be a
+            `hikari.channels.GuildCategory` or the ID of an existing
+            category.
+        reason : hikari.undefined.UndefinedOr[builtins.str]
+            If provided, the reason that will be recorded in the audit logs.
```

Comment:
Do we document the length limit for reason anywhere?

Reply:
No, since discord doesnt document it themselves (if there is, havent seen it).

Reply:
https://github.com/discord/discord-api-docs/commit/e0b3bf3e6499cd9121e9a77af36fff798820bdf9

Reply:
Oh, didnt see that, will add